pkg/task/obcluster_flow.go

```diff
@@ -85,6 +86,9 @@ func UpgradeOBCluster() *TaskFlow {
 			Name:         flowname.UpgradeOBCluster,
 			Tasks:        []string{taskname.ValidateUpgradeInfo, taskname.BackupEssentialParameters, taskname.UpgradeCheck, taskname.BeginUpgrade, taskname.RollingUpgradeByZone, taskname.FinishUpgrade, taskname.RestoreEssentialParameters},
 			TargetStatus: clusterstatus.Running,
+			FailureRule: fail.FailureRule{
```

Comment:
FailureRule can be replaced with Strategy

Reply:
Okey, OnFailure is a better fit.